src/Runtime.php

```diff
@@ -143,12 +143,8 @@ public function templates() {
 	 */
 	public function filesystems() {
 
-		$root = new Filesystem( dirname( $this->plugin_file ) );
-
 		$this->filesystems = [
-			'root'     => $root,
-			'dist'     => new Filesystem( $root->path( 'dist' ) ),
-			'includes' => new Filesystem( $root->path( 'src/includes' ) ),
+			'root' => new Filesystem( dirname( $this->plugin_file ) ),
```

Comment:
We could just do `$this->filesystem = new Filesystem( dirname( $this->plugin_file ) );` instead of carrying the array

Reply:
Okay! Starting dearrayizing...